lib/volapi.js

```diff
@@ -0,0 +1,150 @@
+var util = require('util');
+
+var assert = require('assert-plus');
+
+var RestifyClient = require('./restifyclient');
+
+function VOLAPI(options) {
+    RestifyClient.call(this, options);
+    this.url = options.url;
+}
+
+util.inherits(VOLAPI, RestifyClient);
+
+
+VOLAPI.prototype.close = function close() {
+    this.client.close();
+};
+
+VOLAPI.prototype.createVolume =
+    function createVolume(params, options, callback) {
```

Comment:
Hrm. I'd like to propose that some of us talk (tomorrow?) about picking a preference for function signature for clients, at least for new ones. My preference has moved to wanting `function theName(opts, cb)` for all methods. Here I mean that there are two arguments: the first is options, the second is the callback. I notice that we aren't that consistent (surprise surprise!).

(A separate less important discussion is the what to call those vars: `opts` vs `options`, `callback` vs `cb`.)